consul/share/plugins/consul/30_generic_methods/variable_from_consul.cf

```diff
@@ -39,19 +39,18 @@ bundle agent variable_from_consul(variable_prefix, variable_name, path)
       "promisers" slist => { @{this.callers_promisers}, cf_null }, policy => "ifdefined";
       "class_prefix" string => canonify(join("_", "promisers"));
       "args" slist => {"${variable_prefix}", "${variable_name}", "${path}"};
-
+    
     pass1::
-            "${variable_prefix}.${variable_name}" string => execresult('/usr/bin/curl -s -H "X-Consul-Token: ${consul.config[token]}" http://${consul.config[server_addr]}/v1/kv/${path}?raw', "noshell");
```

Comment:
indentation to fix here too

Reply:
I think i fixed it in the last commit

Reply:
It seems you only fixed the line before but not the line with the command itself.